src/plugins/postgres_table_copy_operator.py

```diff
@@ -112,6 +117,54 @@ def __init__(
                 "even though source table will be dropped."
             )
 
+    def _drop_tables_if_unequal(self, cursor: Any, all_table_copies: List[TableMapping]) -> None:
+        """Drop the target tables that are unequal to their source table."""
+        for table_mapping in all_table_copies:
+            # First make sure the target table exists
+            cursor.execute(
+                sql.SQL("SELECT to_regclass('{target_table_name}')").format(
```

Comment:
I'm very glad to see `psycopg2.sql` used in more and more code in favor of `str` formatting.

Here however a regular DBAPI parameter passing would have sufficed. Eg:

```python
cursor.execute(
    "SELECT to_regclass(%(target_table_name)s)",
    {target_table_name=table_mapping.target}
)
```

Reason being that `target_table_name` is not really an identifier from the SQL grammar point of view. Yes, it is a table name and table names are identifiers, but here it is just a string argument to a PostgreSQL function. Simply using `%(target_table_name)s` will make sure it is quoted as a string argument (hence you can leave the quotes out).

And as we're just using a regular SQL query with plain DBAPI parameters, you can do away with `sql.SQL` and `sql.Identifier`